### ISSUE TITLE
Add public statistics for login view

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-core#1012]
It has been decided we should include publicly visible statistics from apps in the login view for "marketing" purposes.

I will think of the best way to implement this, but I should at least try extending and reusing the stats already included in the `siteinfo` app.

**Comment by holtgrewe** [comment]
Agreed. The display could use some "dashboard tiles" to make the numbers look a bit less dry.

**Comment by mikkonie** [comment]
Agreed, the output needs to be less technical and more casual viewer friendly :)

**Comment by mikkonie** [comment]
We need to discuss this internally so I'm postponing for v1.2 for now.
